inc/pantheon-multisite-finalize.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/*
+ * If a site has multisite enabled, but has not had the final installation
+ * steps completed, alert the user and provide links.
+ */
+
+function pantheon_multisite_install_finalize_message() {
+	?>
+	<div class="notice notice-info is-dismissible">
+		<p>Your WordPress Site Network is almost ready!</p>
```

Comment:
This looks great but all text strings should be internationalized.
See: https://developer.wordpress.com/themes/i18n/